### DISCUSSION TITLE
Imgix as transformer doesn't save transforms/cache in imager imagerSystemPath

### DISCUSSION THREAD

**Discussion by zizther** [spacecatninja/craft-imager-x#158] [General]
When using `imgix` as the `transformer`, are the `imagerSystemPath` and `imagerUrl` setting respected?
I am using craft as the transformer locally and it saves the transforms in the `imager` folder as setup in the config, but on the staging server where it is using `imgix` the transforms appear to be saved in the folder where the assets are instead of the imager folder.

Is this the correct behaviour?

**Comment by aelvan** [comment]
Yes, when using imgix, no images are stored locally, and the url is constructed based on the imgix source url and their api. So yes, `imagerSystemPath` and `imagerUrl` does not come into play and is ignored.
